### PR TITLE
Note about unsupported storage libvirt types

### PR DESCRIPTION
**Pull request by lzap**
More context: https://projects.theforeman.org/issues/33270

### CODE REVIEW

File: guides/common/assembly_provisioning-virtual-machines-kvm.adoc [hunk 1]
```diff
@@ -10,6 +10,9 @@ ifdef::context[:parent-context: {context}]
 Kernel-based Virtual Machines (KVMs) use an open source virtualization daemon and API called `libvirt` running on {RHEL}.
 {ProjectName} can connect to the `libvirt` API on a KVM server, provision hosts on the hypervisor, and control certain virtualization functions.
 
+Only Virtual Machines created through {Project} can be managed.
+Virtual Machines with other than directory storage pool types are unsupported.
```
Comment (melcorr) [inline guides/common/assembly_provisioning-virtual-machines-kvm.adoc:14]:
```suggestion
Virtual machines with other than directory storage pool types are unsupported.
```
Can you help me understand this? 
VMs with external directory storage pool types? 
I can't follow :-(

**Review by melcorr — CHANGES_REQUESTED** [review]

File: guides/common/assembly_provisioning-virtual-machines-kvm.adoc (hunk 1 above)
Comment (lzap) [reply guides/common/assembly_provisioning-virtual-machines-kvm.adoc:14]: Every VM in libvirt usually have a disk (drive), these are created as volumes in storage pools. And there are several storage pool types. Libvirt comes with preconfigured "default" directory pool type and this is the only type the Ruby library Foreman uses supports. Other like LVM pools or physical pools are not supported an will cause an error.

Comment (lzap) [reply guides/common/assembly_provisioning-virtual-machines-kvm.adoc:14]: https://libvirt.org/storage.html

Comment (lzap) [reply guides/common/assembly_provisioning-virtual-machines-kvm.adoc:14]: Are we good? Anything else?

**Comment by lzap** [comment]
Added new commit with your fix.

File: guides/common/assembly_provisioning-virtual-machines-kvm.adoc (hunk 1 above)
Comment (melcorr) [reply guides/common/assembly_provisioning-virtual-machines-kvm.adoc:14]: i have updated my suggestion. i think this is better. As it stands now, it makes no sense.

**Comment by lzap** [comment]
@maximiliankolb any comments? Let's get this merged together! Thanks.

**Comment by lzap** [comment]
Feel free to merge, thanks!

**Pull request merged**